src/graph.ts

```diff
@@ -855,55 +855,58 @@ export class Graph {
             if(matches['matchings'] &&  matches['matchings'].length > 0) {
                 
                 var match = matches['matchings'][0];
-                if(0 < match.confidence ) {
```

Comment:
If OSRM has managed to match points to a route, that's good enough for us. Let's just match whatever the confidence.

For our application, I'd much rather an incorrect match than no match at all.

Reply:
> But I get the point if OSRM is able to give a match then shst should not do further filtering.

Yeah this is maybe a better way of putting it! A match with OSRM should _always_ give a successful output as OSRM is matching to exactly the same network.